src/snowflake/connector/connection.py

```diff
@@ -1182,6 +1182,11 @@ def __config(self, **kwargs):
             ]:
                 self._authenticator = auth_tmp
 
+        # read OAuth token from
+        if "token_file_path" in kwargs:
+            with open(kwargs["token_file_path"]) as f:
+                self._token = f.read()
```

Comment:
will `self._token` be overwritten if there is another authentication method being provided?

Reply:
from the code, it seems self._token is for OAuth only